### DISCUSSION TITLE
csi driver

### DISCUSSION THREAD

**Discussion by gfxlabs** [seaweedfs/seaweedfs-operator#59] [Q&A]
i was looking into using the csi driver in my cluster

the readme says "Compability with [seaweedfs-csi-driver](https://github.com/seaweedfs/seaweedfs-csi-driver)" is not yet done

does that mean if i deploy csi and point it at a seaweedfs cluster running via operator, that it will not function? if so, what work would I need to do? what is incompatible?

**Comment by chrislusf** [comment]
This was to also include csi driver in this operator.
